File: cadastro/WebApplication1/Model/endereco.cs
namespace WebApplication1.Model;

public class endereco
{
    public int Id { get; set; }
    public string TipoEndereco { get; set; }
    public int? Numero { get; set; }
    public string? Complemento { get; set; }
    public string Bairro { get; set; }
    public string Cep { get; set; }
    public string Cidade { get; set; }
    public string Uf { get; set; }

}

File: cadastro/WebApplication1/Front/cadastro-api/src/app/shared/model/endereco.ts
export class Endereco {
    constructor(
        public id?: number,
        public cep?: number,
        public endereco?: string,
        public cidade?: string,
        public numero?: number,
        public complemento?: string,
        public bairro?: string,
        public tipoDoEndereco?: string,
        public uf?: string
    ){}
}